pyproject.toml

```diff
@@ -31,20 +31,20 @@ six = "*"
 docutils = "*"
 "kids.cache" = "*"
 typing = { version = "3.7.*", python = "<3.7" }
+typing-extensions = { version = ">=3.7.4.3", python = ">=3.7,<3.10" }
```

Comment:
I think we want to install typing-extensions for `python = "<3.10"`, since we'll need it in python2 as well. I confirmed that `3.7.4.3` is available for python2.   In which case, the imports will look like this:

```python
if sys.version_info[:2] < (3, 10):
    from typing_extensions import TypedDict
else:
    from typing import TypedDict
```

Reply:
Nice, that makes sense. I should have checked whether typing-extensions was py2 compliant!

I agree that your (major, minor) version is more legible. I can make the change and update the PR tonight.

We're trying to get typing more widely adopted across the studio but our small team at IW has been typing new code for a little over a year (largely influenced, at least on our team, by what I learned at Luma). Most folks at different game studios I've spoken to seem to be interested in adopting static typing and a big obstacle is being able to handle untyped legacy code in a linting/typechecking pre-commit workflow so that's why I'm revisiting typeright. Mostly it would be nice to have it in my back pocket as an auto-typing solution whether professional or personal